apps/music/style/music.css

```diff
@@ -555,13 +555,6 @@ html[dir="rtl"] .tile:nth-child(12n+6) {
   right: 0.5%;
 }
 
-/* Favor scrolling performance over quality while scrolling */
-#views.scroll li.list-item,
```

Comment:
I wonder what this #views.scroll is. I don't see any music app code that sets that class.  Was this a typo in the original patch that added this -moz-crisp-edges stuff, I wonder?